code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -207,7 +207,7 @@
 
 	if(iscarbon(M))
 		var/mob/living/carbon/C = M
-		C.adjustCloneLoss(rand(2,4))
+		C.adjustCloneLoss(rand(2,4) * C.dna.species.clonemod)
```

Comment:
This whole bit should be using `mob/apply_damage()`, which calls `species/apply_damage()` and `clonemod` should be factored into `species/apply_damage()`